### PR TITLE
adding words.text file

### PR DESCRIPTION
**Pull request by jjaijg**
Add this to main

### CODE REVIEW

File: words.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+A new text files contains all the words
```
Comment (Ahmedka114) [inline words.txt:1]: We could add more lines to these sentences

**Comment by Ahmedka114** [comment]
Great

File: words.txt (hunk 1 above)
Comment (jjaijg) [reply words.txt:1]: so added only this comment? I am not able to see any changes in words.txt file

Comment (Ahmedka114) [inline words.txt:1]: change this line

**Review by Ahmedka114 — CHANGES_REQUESTED** [review]

**Pull request merged**